src/socket.js

```diff
@@ -1,43 +1,76 @@
 const socketio = require('socket.io')
 const authMiddleware = require('./../middleware/authenticationSocket')
-const { listPartnerUsers, userTypingStatus } = require('./controllers/message')
+const {
+    listPartnerUsers,
+    userTypingStatus,
+    createMessage,
+} = require('./controllers/message')
+const { default: mongoose } = require('mongoose')
```

Comment:
Nice job Manizha! could you please let me know why do we need mongoose in socket.js?